### PR TITLE
feat: Builtin settings objects from Setup section of Fluent's outline tree

### PR DESCRIPTION
**Pull request by mkundu1**
The classes for built-in settings objects from the Setup section of Fluent's outline tree are generated in a codegen step. Following is an example script using those classes. I'll add the settings classes from the Solution and Results sections of Fluent's outline tree. 

Example script:
```
import ansys.fluent.core as pyfluent
from ansys.fluent.core import Viscous, VelocityInlet, BoundaryConditions
solver = pyfluent.launch_fluent()
solver.file.read_case(file_name=r"d:\work\elbow.cas.h5")
visc = Viscous(solver=solver)
cold_inlet = VelocityInlet(solver=solver, name="inlet2")
bc_group = BoundaryConditions(solver=solver)
```
Static intellisense:

Code_ogPfCT6tQk
Code_Vpl9R2hULv

### CODE REVIEW

File: src/ansys/fluent/core/solver/settings_builtin.pyi
Comment (mkundu1) [inline src/ansys/fluent/core/solver/settings_builtin.pyi:1]: This file is needed for static intellisense. The import paths may not exist if codegen is not run, so we cannot move it to .py.

I haven't added the typing base class for boundary_condition which will have combination of all boundary types.

File: src/ansys/fluent/core/session_solver.pyi
Comment (mkundu1) [inline src/ansys/fluent/core/session_solver.pyi:1]: Couple of issues in static intellisense in the existing code, probably would merge them separately.

File: src/ansys/fluent/core/session_solver.pyi [hunk 1]
```diff
@@ -21,4 +21,4 @@ class Solver:
     def read_case(self, file_name: str): ...
     def write_case(self, file_name: str): ...
     @property
-    def settings(self) -> settings_root: ...
+    def settings(self) -> settings_root.root: ...
```
Comment (mkundu1) [inline src/ansys/fluent/core/session_solver.pyi:24]: Couple of issues in static intellisense in the existing code, probably will merge them separately.

File: src/ansys/fluent/core/solver/settings_builtin.pyi
Comment (mkundu1) [inline src/ansys/fluent/core/solver/settings_builtin.pyi:1]: This file is needed for static intellisense. The import paths may not exist if codegen is not run, so we cannot move this code to .py.

I haven't added the typing base classes for boundary_condition which will be a combination of all boundary types.

File: src/ansys/fluent/core/solver/settings_builtin_data.py
Comment (mkundu1) [inline src/ansys/fluent/core/solver/settings_builtin_data.py:1]: To add new built-in settings, we need to update this file. The settings classes (and type-stubs) are generated as a part of codegen.

**Review by hpohekar — APPROVED** [review]

**Review by seanpearsonuk — APPROVED** [review]

**Review by prmukherj — APPROVED** [review]

**Pull request merged**